frontend/src/routes/Applications/CreateApplication/Subscription/common/TimeWindow/index.js

```diff
@@ -69,8 +69,8 @@ export class TimeWindow extends Component {
         ])
     }
 
-    validation(exceptions) {
-        const { control, i18n } = this.props
+    validation(exceptions, i18n) {
+        const { control } = this.props
```

Comment:
Bad revert?

Reply:
yes, should be referenced from props. fixing.